File: Pemrograman Berorientasi Objek/Materi/Pertemuan 3/Inheritance/cpp/main.cpp
#include <string>
#include <iostream>
using namespace std;
#include "Manusia.cpp"
#include "Karyawan.cpp"

int main(){
	
	Manusia kmanusia;
	Karyawan kkaryawan;
	
	kmanusia.setNama("Gina");
	kmanusia.setAlamat("Bandung");
	kmanusia.setNoKTP("320.120.777");
	kmanusia.setNoTelepon("08188899977");
	
	cout << kmanusia.getNama() << endl;
	cout << kmanusia.getAlamat() << endl;
	cout << kmanusia.getNoKTP() << endl;
	cout << kmanusia.getNoTelepon() << endl;
	
	kkaryawan.setNama("Gina");
	kkaryawan.setAlamat("Bandung");
	kkaryawan.setNoKTP("320.120.777");
	kkaryawan.setNoTelepon("08188899977");
	kkaryawan.setNomorPegawai("13202020");
	kkaryawan.setJabatan("-");
	kkaryawan.setDepartemen("-");
	
	cout << kkaryawan.getNama() << endl;
	cout << kkaryawan.getAlamat() << endl;
	cout << kkaryawan.getNoKTP() << endl;
	cout << kkaryawan.getNoTelepon() << endl;
	cout << kkaryawan.getNomorPegawai() << endl;
	cout << kkaryawan.getJabatan() << endl;
	cout << kkaryawan.getDepartemen() << endl;
	
	return 0;
}

File: Pemrograman Berorientasi Objek/Materi/Pertemuan 3/Inheritance/java/Main.java
class Main{
	
	public static void main(String[] args){
		
		Manusia kmanusia = new Manusia();
		Karyawan kkaryawan = new Karyawan();
		
		kmanusia.setNama("Gina");
		kmanusia.setAlamat("Bandung");
		kmanusia.setNoKTP("320.120.777");
		kmanusia.setNoTelepon("08188899977");
		
		System.out.println(kmanusia.getNama());
		System.out.println(kmanusia.getAlamat());
		System.out.println(kmanusia.getNoKTP());
		System.out.println(kmanusia.getNoTelepon());
	
		kkaryawan.setNama("Gina");
		kkaryawan.setAlamat("Bandung");
		kkaryawan.setNoKTP("320.120.777");
		kkaryawan.setNoTelepon("08188899977");
		kkaryawan.setNomorPegawai("13202020");
		kkaryawan.setJabatan("-");
		kkaryawan.setDepartemen("-");
		
		System.out.println(kkaryawan.getNama());
		System.out.println(kkaryawan.getAlamat());
		System.out.println(kkaryawan.getNoKTP());
		System.out.println(kkaryawan.getNoTelepon());
		System.out.println(kkaryawan.getNomorPegawai());
		System.out.println(kkaryawan.getJabatan());
		System.out.println(kkaryawan.getDepartemen());
		
	}
	
}